registry/storage/driver/base/base.go

```diff
@@ -42,9 +42,21 @@ import (
 	"io"
 
 	dcontext "github.com/docker/distribution/context"
+	prometheus "github.com/docker/distribution/metrics"
 	storagedriver "github.com/docker/distribution/registry/storage/driver"
+	"github.com/docker/go-metrics"
+	"time"
 )
 
+var (
+	// storageAction is the metrics of blob related operations
+	storageAction = prometheus.StorageNamespace.NewLabeledTimer("storage_action", "The number of seconds that the storage action takes", "driver", "action")
```

Comment:
This stutters: it becomes `registry_storage_storage_action`. Should just be `action` here.